edge-apache/development-edge-httpd.conf

```diff
@@ -66,6 +66,8 @@ LoadModule ssl_module modules/mod_ssl.so
 
 
 <VirtualHost *:443>
+    TimeOut 300
```

Comment:
The default Apache 2.4 timeout is 60 seconds, which can kill long-running workflows. We set it to 300 seconds here, but we may need to tweak that depending on how long the actual UMAP stuff runs. Note that there is also the same setting that should be adjusted for the `app_fe` containers.